packages/x-scheduler-headless/src/event-calendar-selectors/eventCalendarAgendaSelectors.ts

```diff
@@ -15,6 +15,7 @@ export const eventCalendarAgendaSelectors = {
   visibleDays: createSelectorMemoized(
     (state: State) => state.adapter,
     schedulerOtherSelectors.visibleDate,
+    schedulerOtherSelectors.uiTimezone,
```

Comment:
Not for this PR: I saw that Mobiscroll called it `displayTimezone`, I think I like it more than `uiTimezone`.
They also use `props.displayTimezone`. The added explicitness can be nice as well.
WDYT?